clusters/overlays/rhoai-stable-2.10/patch-application-repo-revision.yaml

```diff
@@ -0,0 +1,10 @@
+- op: replace
+  path: /spec/source/repoURL
+  # TODO Change back when ready to merge back to upstream.
+  # value: 'https://github.com/redhat-ai-services/ai-accelerator.git'
+  value: 'https://github.com/fgharo/ai-accelerator.git'
```

Comment:
Can you change this back to the redhat-ai-services/ai-accelerator.git

Reply:
Yup. Done.